src/hooks/useArticleTextSize.js

```diff
@@ -1,12 +1,18 @@
-import { useState, useLayoutEffect } from 'preact/hooks'
+import { useState, useEffect } from 'preact/hooks'
 import { articleTextSize } from 'utils'
 
-export const useArticleTextSize = (dependencies = []) => {
+let setArticleTextSize

```

Comment:
State like that should be avoided as much as possible in hooks. You'll need a good reason and a good comment to pass code review.

Reply:
So the reason here is the instantaneous update of the article font size (in the background) as the user clicks left or right in the `TextSize` component with the slider. I tried a couple of different ideas but the behavior always had a delay. That is, the background text wouldn't update instantly and instead update in the next render once the user closed `TextSize` (or even worse after scrolling a bit in the article itself). High level ideas I tried:

* an extra `useEffect` in `ArticleSection` to trigger an `articleTextSize.adjust(0)`, with and without dependencies
* a weird idea that didn't get anywhere was setting the `useArticleTextSize` hook in `ArticleInner`, one level up so that it could be passed down to both `TextSize` and `ArticleSection` and hence use the same state. But alas that didn't work. 
* and then tried different combinations of `useEffect` hooks in `useArticleTextSize`, with and without dependencies, but the only way I was able to get the instant article text update was with that `setArticleTextSize`

Does this spark any better ideas we can try? Idea is to somehow link `ArticleSection` and `TextSize` to achieve the instant update in the background.